evalai/utils/challenges.py

```diff
@@ -294,3 +293,47 @@ def display_challenge_phase_detail(challenge_id, phase_id):
 
     phase = response
     pretty_print_challenge_phase_data(phase)
+
+
+def pretty_print_challenge_phase_split_data(splits):
+    """
+    Function to print the details of a Challenge Phase Split.
+    """
+    for split in splits:
+        br = style("----------------------------------------"
+                   "--------------------------", bold=True)
+
+        dataset_split_name = "{}".format(style(split["dataset_split_name"], bold=True, fg="green"))
+        dataset_split_id = "ID: {}\n\n".format(style(str(split["dataset_split"]), bold=True))
```

Comment:
One thing I noticed that we are displaying wrong ID here:
It should be 
`challenge_phase_split_id = "Challenge Phase Split Id: {}\n\n".format(style(str(split["id"]), bold=True))`